common.mk

```
@@ -1298,7 +1298,7 @@ OPTS =
 # See benchmark/README.md for details.
 benchmark: miniruby$(EXEEXT) update-benchmark-driver PHONY
 	$(BASERUBY) -rrubygems -I$(srcdir)/benchmark/lib $(srcdir)/benchmark/benchmark-driver/exe/benchmark-driver \
-	            --executables="compare-ruby::$(COMPARE_RUBY) -I$(EXTOUT)/common --disable-gem" \
+	            --executables="compare-ruby::$(COMPARE_RUBY) -I$(EXTOUT)/common --disable-gem $(RUN_OPTS)" \
 	            --executables="built-ruby::$(BENCH_RUBY) --disable-gem" \
```

```suggestion
	            --executables="built-ruby::$(BENCH_RUBY) --disable-gem $(RUN_OPTS)" \
```

What about the `--executables` of built-ruby? If you're comparing multiple binaries, it's likely that you're not interested in interp vs yjit. So I think the same RUN_OPTS should be passed to built-ruby as well.